theme/static/css/nwcpp.css

```diff
@@ -16,17 +16,19 @@
 }
 
 .content-wrapper {
-	width: 1100px;
+	width: 1200px;
 	margin: 0 auto;
 }
 
 .sidebar {
-	float: right;
-	width: 280px;
 	border-radius: 10px;
 	background: #fff;
 	margin-bottom: 2em;
 	padding: 5px;
+    left:85%;
```

Comment:
Can we fix the indentation here?  The new lines seem to me to be further to the left than the lines left from before.  Maybe a tabs vs spaces issue?

Reply:
since we are using absolute positioning, we want the third column to start at 85% from the lhs